internal/typeinfo/member.go

```diff
@@ -116,3 +150,17 @@ func (f *structField) GetScanTarget(outVal reflect.Value) (any, *ScanProxy, erro
 	}
 	return val.Addr().Interface(), nil, nil
 }
+
+// locateValue locates the value corresponding to the given type in the
+// typeToValueMap and returns an error message if it cannot be found.
+func locateValue(typeToValue map[reflect.Type]reflect.Value, typ reflect.Type) (reflect.Value, error) {
```

Comment:
Same comment, if we could change this function to report the error and we can have the map access directly in the `LocateScanTarget` functions, that would make it simpler and easier to read.

Reply:
Changed.